settings.gradle.kts

```diff
@@ -1,2 +1,11 @@
-// Empty file preventing Intellij from freaking out
-rootProject.name = "forma"
+pluginManagement {
```

Comment:
Let's don't do that. I want to avoid having one mega project in this repo. 
Currently, srttings.gradle.kts is in place to prevent IDEA from freaking out because it can't detect the project type.

The repository operates just fine by having multiple separate projects which uses `includeBuild` functionality. Any contributor may just open the project it wants to contribute to and save some time waiting for IDE sync